app/src/main/java/io/intrepid/contest/utils/ImageUtils.java

```diff
@@ -0,0 +1,18 @@
+package io.intrepid.contest.utils;
+
+import android.graphics.Bitmap;
+import android.util.Base64;
+
+import java.io.ByteArrayOutputStream;
+
+public class ImageUtils {
+    private static final Bitmap.CompressFormat FORMAT = Bitmap.CompressFormat.PNG;
+    private static final int QUALITY = 100;
```

Comment:
The quality param in the `compress` method is an int from 0 to 100, but I couldn't find an android constant for this...
https://developer.android.com/reference/android/graphics/Bitmap.html
And actually our API currently has a 3MB limit, so in the future I might have to compress it until it gets to 3MB, or just tell the user to choose a smaller image